src/components/TableRow.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const TableRow = ({ link, title, subtitle }) => (
```

Comment:
Generalize this table row component, so it only gets the data to render instead of needing to know the data contract